### PR TITLE
Fix Kotlin 1.4 runtime error on measure time

### PR DESCRIPTION
**Pull request by jeziellago**
Fixing runtime errors with kotlin 1.4  (#847 #871)

### CODE REVIEW

**Comment by arnaudgiuliani** [comment]
Before using this over deprecated API, I was just timing by myself. Do you see any good way without using an unstable API?

**Comment by jeziellago** [comment]
> Before using this over deprecated API, I was just timing by myself. Do you see any good way without using an unstable API?

@arnaudgiuliani I rewrote without `kotlin.time`. Could you check it?

File: koin-projects/koin-core/src/main/kotlin/org/koin/core/time/Measure.kt [hunk 1]
```diff
@@ -28,15 +25,12 @@ import kotlin.time.TimeSource
...
  * Measure code execution
  */
-@OptIn(ExperimentalTime::class)
 fun measureDuration(code: () -> Unit): Double {
-    val clock = TimeSource.Monotonic
-    val mark = clock.markNow()
+    val initialTime = System.nanoTime().toDouble()
     code()
-    return mark.elapsedNow().inMilliseconds
+    return (System.nanoTime() - initialTime).inMilliseconds()
```
Comment (arnaudgiuliani) [inline koin-projects/koin-core/src/main/kotlin/org/koin/core/time/Measure.kt:31]: just simple extract as `startTime` & `endTime` those variables 👍

File: koin-projects/koin-core/src/main/kotlin/org/koin/core/time/Measure.kt [hunk 2]
```diff
@@ -45,17 +39,17 @@ fun measureDuration(message: String, code: () -> Unit) {
...
  * Measure code execution and get result
  */
-@OptIn(ExperimentalTime::class)
 fun <T> measureDurationForResult(code: () -> T): Pair<T, Double> {
-    val clock = TimeSource.Monotonic
-    val mark = clock.markNow()
+    val initialTime = System.nanoTime().toDouble()
     val result = code()
-    return Pair(result, mark.elapsedNow().inMilliseconds)
+    val elapsedTime = (System.nanoTime() - initialTime).inMilliseconds()
```
Comment (arnaudgiuliani) [inline koin-projects/koin-core/src/main/kotlin/org/koin/core/time/Measure.kt:45]: just simple extract as `startTime` & `endTime` those variables 👍

Comment (ppjeziellago) [reply koin-projects/koin-core/src/main/kotlin/org/koin/core/time/Measure.kt]: Done! 🚀

**Pull request merged**